docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -357,8 +357,8 @@ The server will need an access to public Internet via IPv4 (at the time of writi
 
 Keys:
 
--   `enabled` – Boolean flag to enable the feature. Set to `true` to allow sending crash reports.
--   `endpoint` – Overrides the Sentry endpoint.
+-   `enabled` – Boolean flag to enable the feature, `false` by default. Set to `true` to allow sending crash reports. 
+-   `endpoint` – You can Overrides the Sentry endpoint URL for sending crash reports and use your self-hosted Sentry installation. Use [Sentry DSN](https://docs.sentry.io/error-reporting/quickstart/?platform=native#configure-the-sdk) syntax. 
```

Comment:
```suggestion
-   `endpoint` – You can override the Sentry endpoint URL for sending crash reports. It can be either separate Sentry account or your self-hosted Sentry instance. Use the [Sentry DSN](https://docs.sentry.io/error-reporting/quickstart/?platform=native#configure-the-sdk) syntax. 
```